committee-steering/governance/sig-governance-template-long.md

```diff
@@ -0,0 +1,449 @@
+# SIG ~~Foo~~ Governance
+
+The following document outlines the governance for
+SIG ~~Foo~~.  Its goals are to define the necessary structure for
+SIG ~~Foo~~ to self govern and to define leadership responsibilities
+within the SIG.
+
+This charter may be updated the members of SIG foo through lazy
+consensus.
+
+This document is broken up into the following parts:
+
+- Roles within the SIG and their responsibilities
+- Processes used by members of the SIG
+
+## Roles and responsibilities
+
+This section describes the various roles governing the SIG.
+
+### Sig leads
```

Comment:
Sheppard, Function Lead, Manager ... But not Lead IMO.  

/cc @jbeda

Reply:
Shepherd

Steward

Facilitator

Admin

Coordinator

Chair

Whip

Reply:
of those listed, if Lead is not desired, I guess Admin or Administrator is preferred.